docs/install-cpu-source.rst

```diff
@@ -5,7 +5,7 @@
 
     # CPU version of pytorch has smaller footprint - see installation instructions in
     # pytorch documentation - https://pytorch.org/get-started/locally/
-    pip3 install torch==1.12.1+cpu torchvision==0.13.1+cpu torchtext==0.13.1 --extra-index-url https://download.pytorch.org/whl/cpu
+    pip3 install torch==1.13.1+cpu torchvision==0.14.1+cpu --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
Reminder: Please also update `stable` and `0.7.0` branches with this PR's changes

Reply:
that's the plan. Thanks for reminder